dydx3/modules/private.py

```diff
@@ -163,6 +164,9 @@ def update_user(
         :param country optional
         :type country: str (ISO 3166-1 Alpha-2)
 
+        :param language_code optional
+        :type language_code: str (ISO 639-1, including 'zh-CN')
```

Comment:
Why do we need to call out 'zh-CN' specifically/

Reply:
ISO 639-1 officially only allows `zh` but if you search online for ISO 639-1 codes almost all of them will have `zh-CN` (distinguished from `zh-HK` and `zh-TW`) because that's an IETF language tag.